### DISCUSSION TITLE
The script is not working

### DISCUSSION THREAD

**Discussion by jekobpl** [ej52/proxmox-scripts#93] [Q&A]
[success] Installing dependencies
[success] Installing python
[success] Installing openresty
[success] Installing nodejs
[success] Checking for latest NPM release
[error] Downloading NPM v.tar.gz
/tmp/npm_install.sh: line 113: 'wget $WGETOPT -c $NPMURL/archive/v$_latest_version.tar.gz -O - | tar -xz'

unexpected end of file
Child returned status 1
exiting now

**Comment by Eeems** [comment]
https://github.com/ej52/proxmox-scripts/issues/88
